.build/cspell-words.txt

```diff
@@ -1,16 +1,21 @@
 adfs
 Adsi
 amsi
-Antispam
+Andriod
```

Comment:
This is incorrect spelling remove from file to verify that the correct spelling isn't going to trigger this process.